core/src/main/java/org/bitcoinj/evolution/SimplifiedMasternodeList.java

```diff
@@ -49,16 +50,10 @@ public SimplifiedMasternodeList(NetworkParameters params) {
     SimplifiedMasternodeList(SimplifiedMasternodeList other, short version) {
         super(other.params);
         this.version = version;
-        this.protocolVersion = version == SimplifiedMasternodeListDiff.BASIC_BLS_VERSION ?
-                NetworkParameters.ProtocolVersion.CURRENT.getBitcoinProtocolVersion() :
-                NetworkParameters.ProtocolVersion.BLS_LEGACY.getBitcoinProtocolVersion();
+        this.protocolVersion = params.getProtocolVersionNum(NetworkParameters.ProtocolVersion.SMNLE_VERSIONED);
         this.blockHash = other.blockHash;
         this.height = other.height;
         mnMap = new HashMap<Sha256Hash, SimplifiedMasternodeListEntry>(other.mnMap);
-        // tell all entries to render as version 2 (BLS basic scheme)
-        for (SimplifiedMasternodeListEntry mn : mnMap.values()) {
-            mn.version = version;
-        }
```

Comment:
changes in Core 19.2 no longer require updating the version of masternode list entries